### PR TITLE
Add sandbox projects to enable quick start with the project

### PR DESCRIPTION
**Pull request by derberg**
**Description**

Changes proposed in this pull request:

- Add sandbox projects to enable quick start with the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ Run this command to install the component:
   yarn add @kyma-project/documentation-component
   ```
 
+## Quick start
```
Comment (kazydek) [inline README.md]:
```suggestion
### Quick start
```

File: README.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ Run this command to install the component:
...
+
+To quickly start playing with Documentation Component, use one of below sandboxes:
```
Comment (kazydek) [inline README.md]:
```suggestion
To quickly start playing with the Documentation component, use one of these sandboxes:
```

File: README.md [hunk 3]
```diff
@@ -23,6 +23,14 @@ Run this command to install the component:
...
+
+[![Edit dc-with-all-render-engines](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/derberg/dc-demo-all-render-engines/tree/master/?fontsize=14) - All the render engines in action with navigation for multiple markdown files.
+
+[![Edit dc-with-nav-and-metadata-support](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/derberg/dc-demo-nav-with-metadata-support/tree/master/?fontsize=14) - Plugins in action shown on markdown render engine and support for rendering content from frontmatter and extending navigation with it.
```
Comment (kazydek) [inline README.md]:
```suggestion
[![Edit dc-with-nav-and-metadata-support](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/derberg/dc-demo-nav-with-metadata-support/tree/master/?fontsize=14) - Plugins in action shown on the Markdown render engine. Support for rendering content from front matter and extending the UI navigation with it.
```

File: README.md [hunk 4]
```diff
@@ -23,6 +23,14 @@ Run this command to install the component:
   yarn add @kyma-project/documentation-component
   ```
 
+## Quick start
+
+To quickly start playing with Documentation Component, use one of below sandboxes:
+
+[![Edit dc-with-all-render-engines](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/derberg/dc-demo-all-render-engines/tree/master/?fontsize=14) - All the render engines in action with navigation for multiple markdown files.
```
Comment (kazydek) [inline README.md]:
```suggestion
[![Edit dc-with-all-render-engines](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/derberg/dc-demo-all-render-engines/tree/master/?fontsize=14) - All the render engines in action, with navigation for multiple Markdown files.
```

Comment (kazydek) [reply README.md]: Navigation for multiple Markdown files? 🤔 What do you mean exactly?

File: README.md (hunk 1 above)
Comment (derberg) [reply README.md]: why? it is separate section and not a subsection of installation

Comment (kazydek) [reply README.md]: I added it to be compliant with the template we use. Perhaps you can add it as a subsection to a different section if it fits better somewhere else.

**Pull request merged**